tests/unit/test_load_data.py

```diff
@@ -12,23 +12,28 @@
 
 
 def test_check_if_s3_image_exists_good_filename():
-    assert check_if_s3_image_exists(VALID_FITS_FILENAME)
+    s3_directory = "data"
+    assert check_if_s3_image_exists(VALID_FITS_FILENAME, s3_directory)
 
 
 def test_check_if_s3_image_exists_bad_filename():
-    assert not check_if_s3_image_exists(NONEXISTANT_FILE)
+    s3_directory = "data"
+    assert not check_if_s3_image_exists(NONEXISTANT_FILE, s3_directory)
```

Comment:
Super minor detail, but NONEXISTENT is spelled wrong throughout this file

Reply:
Oops, that's embarrassing! Good find.